### DISCUSSION TITLE
Limit on number of Codespaces running

### DISCUSSION THREAD

**Discussion by bartr** [community/community#8934] [Codespaces]
We are getting an error when we try to create our 3rd Codespace. Is there any way for us to increase this limit?

We are on the paid Teams plan.

**Comment by shenanigans77-jpg** [comment]
What I thinking is to get any thing to work properly is to start from the beginning. Since I'm totally new to this its so far been woekung in my behalf I think.so I seek out for the oldest unfinished item to syce better.i thinking .also..if I was an artificial intelligence system I figure that it automatically would work in that matter it Self due to its a man skilled operation data made. So to remember to stay numerically order as well as in alphabet order .on the code listing.

Sent from Yahoo Mail on Android 
 
  On Thu, Dec 16, 2021 at 12:07 AM, Bart ***@***.***> wrote:   


We are getting an error when we try to create our 3rd Codespace. Is there any way for us to increase this limit?

We are on the paid Teams plan.
